airbyte-webapp/src/components/connectorBuilder/Builder/BuilderSidebar.module.scss

```diff
@@ -0,0 +1,102 @@
+@use "scss/variables";
+@use "scss/colors";
+@use "scss/mixins";
+
+.container {
+  padding: variables.$spacing-sm;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+}
+
+.connectorImg {
+  @include mixins.shadow;
+
+  width: 90px;
+  border-radius: 25px;
+  padding: variables.$spacing-xl;
+  margin-top: 55px;
+}
+
+.connectorName {
+  margin-top: variables.$spacing-md;
+  min-height: 28px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  overflow-x: auto;
+}
+
+.connectorNameText {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.streamsHeader {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 0;
+  margin-top: variables.$spacing-2xl;
+}
+
+.streamsHeading {
+  margin-left: variables.$spacing-md;
+  color: colors.$blue;
+}
+
+.addStreamButton {
+  margin-left: auto;
+  margin-right: variables.$spacing-sm;
+}
+
+.downloadButton {
```

Comment:
It seems like this rule (and the whole div around the button itself) is only needed to push the buttons to the bottom of the screen.

The more idiomatic way of doing this is to give the element in the flex container you want to take the remaining size a `flex-grow: 1` (in this case it would be the stream list). This is how it looks like:
<img width="954" alt="Screenshot 2022-12-09 at 15 12 38" src="https://user-images.githubusercontent.com/1508364/206721210-c8c81abd-76b8-4eb1-8046-7e8231ea94e1.png">

Reply:
Thanks for the suggestion, updated and I agree that feels like a better approach